roles/tests/server.js

```diff
@@ -27,7 +27,7 @@ describe('roles', function () {
       return
     }
 
-    return Meteor.roleAssignment.find({ _id: loggedInUserId })
+    return Meteor.roleAssignment.find( { _id: loggedInUserId });
```

Comment:
At least the added semicolon should not pass the check for code-formatting. Please also evaluate it for the rest of the changes.